server/user_account/models.py

```diff
@@ -32,6 +33,7 @@ def get_role(self):
 
     @receiver(post_save, sender=settings.AUTH_USER_MODEL)
     def create_auth_token(sender, instance=None, created=False, **kwargs):
+        # pylint: disable-all
```

Comment:
I disable all here because the method does not seem complete and will probably be modified later. 
Here are the errors:
user_account/models.py:35:4: E0213: Method should have "self" as first argument (no-self-argument)
user_account/models.py:35:0: W0613: Unused argument 'kwargs' (unused-argument)
user_account/models.py:35:4: R0201: Method could be a function (no-self-use)

Reply:
Then can you add a todo so that we don't forget?